src/assets/images/Pin.svg

```diff
@@ -1 +1,3 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 54.03 81.29"><defs><style>.cls-1{fill:none;stroke:#666;stroke-miterlimit:10;stroke-width:2px;}</style></defs><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><path class="cls-1" d="M53,27c.5-15.5-11.64-26-26-26S.51,11.5,1,27c.48,14.58,25.5,52.5,25.5,52.5S52.54,41.69,53,27Z"/><circle class="cls-1" cx="27.51" cy="26.5" r="10"/></g></g></svg>
```

Comment:
weird this file was removed and renamed to `pin.svg` w/a lowercase p.. will check on this